### ISSUE TITLE
Releases

### ISSUE THREAD

**Issue by gpucce** [JuliaAnimators/JavisNB.jl#10]
@JuliaRegistrator register()

**Comment by gpucce** [comment]
@Wikunia @TheCedarPrince I should have registered the package. Note that right now test fails because I tagged Javis as 0.8.0 which will be the release that will work with this package. Hope this is how it is supposed to go.

**Comment by gpucce** [comment]
Oh and also thank you both guys for all the help @Wikunia and @TheCedarPrince!!!

**Comment by gpucce** [comment]
@Wikunia @TheCedarPrince 
So the auto merge thing
wants us to to add the .jl at the end of the repo name but I believe I can't do it. 

Also there are some issues with the fact that the compat for Javis is currently too high so the automatic merge in the registry can't happen. I can either remove the compat for now or you can push Javis to 0.8 and then I will retrigger the registration.

Let me know what you prefer!

**Comment by Wikunia** [comment]
Oh yeah we should have named it JavisNB.jl 
Hmm interesting case with the version. I don't want to Release v0.8 of Javis when JavisNB isn't tagged as we would still need to wait for 3 days. 
The test cases will fail though if we run JavisNB with the current version of Javis, right? Which probably also breaks auto merging. Also tagging JavisNB with compat Javis pre 0.8 we would have a version that someone might install and expect it to work with Javis pre 0.8 😂

**Comment by Wikunia** [comment]
I have renamed the repo. Do you mind changing the links @gpucce ? Mostly in docs maybe readme

**Comment by gpucce** [comment]
> Oh yeah we should have named it JavisNB.jl Hmm interesting case with the version. I don't want to Release v0.8 of Javis when JavisNB isn't tagged as we would still need to wait for 3 days. The test cases will fail though if we run JavisNB with the current version of Javis, right? Which probably also breaks auto merging. Also tagging JavisNB with compat Javis pre 0.8 we would have a version that someone might install and expect it to work with Javis pre 0.8 😂

On the issue in the registry it says we can comment and explain and then it will be merged manually. If this is the way I will write the comment where I explain the situation

**Comment by Wikunia** [comment]
Yeah that would be good. Make sure to update the links first though and then trigger the register command again.

**Comment by gpucce** [comment]
> Yeah that would be good. Make sure to update the links first though and then trigger the register command again.

I think they were all already set with JavisNB.jl already.

**Comment by Wikunia** [comment]
Looks like it yes.

**Comment by gpucce** [comment]
@JuliaRegistrator register()

**Comment by Wikunia** [comment]
@JuliaRegistrator register()

**Comment by Wikunia** [comment]
@JuliaRegistrator register()

**Comment by Wikunia** [comment]
@JuliaRegistrator register()
